diracx-routers/tests/jobs/test_status.py

```diff
@@ -2,6 +2,7 @@
 
 from datetime import datetime, timedelta, timezone
 from http import HTTPStatus
+from time import sleep
```

Comment:
Can you open an issue to remove sleeps from the tests?

Reply:
#519